src/runtime/pay-client.js

```diff
@@ -222,7 +223,15 @@ export class PayClient {
       )
       .catch(reason => {
         if (typeof reason == 'object' && reason['statusCode'] == 'CANCELED') {
-          return Promise.reject(createCancelError(this.win_));
+          const error = createCancelError(this.win_);
+          if (this.request_) {
```

Comment:
Use the request param instead of this.request_.

Reply:
Also fixed!